src/node_api.cc

```diff
@@ -718,7 +740,7 @@ void napi_module_register_cb(v8::Local<v8::Object> exports,
 
   // Create a new napi_env for this module. Once module unloading is supported
   // we shall have to call delete on this object from there.
-  napi_env env = new napi_env__(context->GetIsolate());
+  napi_env env = v8impl::get_env(context->GetIsolate());
```

Comment:
The comment should probably change now as we'd no longer delete the env when a module is unloaded.